.github/workflows/continuous-deployment.yml

```diff
@@ -860,79 +866,48 @@ jobs:
-        if: always()
-        run: |
-          name="gha-${{ env.priority }}-${{ env.stackName }}"
-          ops/scripts/pipeline/slots/dev-rm-allowed-ip.sh ${{ steps.rgApp.outputs.out }} ${{ env.stackName }} ${name:0:32} ${{ env.slotName }}
-
-  playwright-e2e-test:
-    needs: [build-info, deploy-webapp, deploy-service]
-    env:
-      stackName: ${{ needs.build-info.outputs.stackName }}
-      webappName: ${{ needs.build-info.outputs.webappName }}
-      priority: 201
-    defaults:
-      run:
-        working-directory: test/e2e
-    runs-on: ubuntu-latest
-    steps:
-      - name: Get source code
-        uses: actions/checkout@v3
-      - uses: azure/login@v1
-        with:
-          creds: ${{ secrets.AZURE_CREDENTIALS }}
-          environment: ${{ vars.AZURE_ENVIRONMENT }}
-      - uses: cloudposse/github-action-secret-outputs@main
-        id: rgApp
-        with:
-          secret: ${{ secrets.PGP_SIGNING_PASSPHRASE }}
-          op: decode
-          in: ${{ needs.build-info.outputs.azResourceGrpAppEncrypted }}
-      - name: Enable GHA runner access
+      - name: Seed e2e database
         run: |
-          ../../ops/scripts/pipeline/dev-add-allowed-ip.sh \
-          -g ${{ steps.rgApp.outputs.out }} \
-          -s ${{ env.stackName }} \
-          -p ${{ env.priority }} \
-          --is-cicd
-      - name: Install Node dependencies
+          curl -v -d '{"txIdOverride": '0'}' -H "Content-Type: application/json" https://${{ env.apiName }}.azurewebsites${{ vars.AZ_HOSTNAME_SUFFIX }}/api/orders-sync?x-ms-routing-name=${{ env.slotName }}
+      - name: Execute Playwright Integration test
         run: |
+          pushd test/e2e
           npm ci
           npx playwright install chromium msedge
-      - name: Run the test
-        run: |
           TARGET_HOST=https://${{ env.webappName }}.azurewebsites${{ vars.AZ_HOSTNAME_SUFFIX }} npx playwright test --reporter=list,html
+          popd
       - name: Upload report
         uses: actions/upload-artifact@v4
         if: always()
         with:
           name: playwright-report
           path: test/e2e/playwright-report/
           retention-days: 30
+
       - name: Disable GHA runner access
         if: always()
         run: |
           name="gha-${{ env.priority }}-${{ env.stackName }}"
-          ../../ops/scripts/pipeline/dev-rm-allowed-ip.sh ${{ steps.rgApp.outputs.out }} ${{ env.stackName }} ${name:0:32}
+          ops/scripts/pipeline/slots/dev-rm-allowed-ip.sh ${{ steps.rgApp.outputs.out }} ${{ env.stackName }} ${name:0:32} ${{ env.slotName }}
+
+      - name: Clean up e2e test database
+        if: always()
+        run: |
+          e2eDatabaseName="${{ secrets.AZ_COSMOS_DATABASE_NAME }}-e2e"
+          if [[ ${{ vars.ENVIRONMENT }} != "Main-Gov" ]]; then
+            echo "Adding suffix to database name"
+            e2eDatabaseName="${e2eDatabaseName}-${{ env.branchHashId }}"
+          fi
+          az cosmosdb sql database delete \
```

Comment:
This delete command works locally. Need to troubleshoot whether the logic for generating e2eDatabaseName is correct.

Reply:
Fixed. Used wrong variable name.